app/views/entities/index.html.erb

```diff
@@ -1,22 +1,28 @@
 <p style="color: green"><%= notice %></p>
 <h1 class="title is-3 has-text-centered mt-5">Transaction</h1>
-<div id="entities" class="box"> <% @entities.each do |entity| %>
+<%= link_to "Back", categories_path, class: "button is-danger is-rounded is-small" %>
+<div id="entities" class="box">
+  <% @entities.each do |entity| %>
     <div class="media m-6">
       <div class="media-left">
-        <figure class="image is-64x64 margin-4"> <img src="https://cdn.onlinewebfonts.com/svg/img_218090.png" alt="image">
+        <figure class="image is-64x64 margin-4">
+          <img src="https://cdn.onlinewebfonts.com/svg/img_218090.png" alt="image">
         </figure>
       </div>
-      <div class="media-content"> <strong><%= entity.name %><strong>
-            <div class="has-text-right"> <span>$<span><%= entity.amount %> </div>
-              </div>
-            </div>
-            <p class="ml-6"><%= entity.created_at.strftime("%d %h %Y")%> at <%= entity.created_at.strftime("%l.%M%P")%>
-</p>
-          <% end %>
-        </div>
-        <div class="buttons is-centered"> <%= link_to "Add New Transaction", new_entity_path, class: "button is-link is-light is-rounded is-normal" %>
+      <div class="media-content">
+        <strong><%= entity.name %></strong>
+        <div class="has-text-right">
+          <span>$<span><%= entity.amount %> </div>
+          </div>
         </div>
+        <p class="ml-6">
+          <%= entity.created_at.strftime("%d %h %Y")%> at <%= entity.created_at.strftime("%l.%M%P")%>
+        </p>
       </div>
+    <% end%>
+    <div class="buttons is-centered">
+      <%= link_to "Add New Transaction", new_category_entity_path(@category.id), class: "button is-link is-light is-rounded is-normal" %>
     </div>
   </div>
 </div>
+</div>
```

Comment:
- [ ] Kindly style the entities page based on the design.

- [ ] At the top of the transactions page there should be a notice indicating the total amount of the expenses as we have in the design.

- [ ] Your transactions should be ordered by the most recent. That means the last transactions should be listed first.

  Your Implementation | Behance Design
  :---:|:---:
  ![rior](https://user-images.githubusercontent.com/54848363/214604571-c15a0c22-1e18-4bb5-8d46-571d626a258f.jpg) | ![riew](https://user-images.githubusercontent.com/54848363/214604582-a25c3b79-91ab-4256-882a-fb35c6991603.jpg)